contribs/drt-extensions/src/main/java/org/matsim/contrib/drt/extension/edrt/optimizer/EDrtVehicleDataEntryFactory.java

```diff
@@ -74,7 +74,11 @@ public VehicleEntry create(DvrpVehicle vehicle, double currentTime) {
 		if (schedule.getStatus() == ScheduleStatus.PLANNED) {
 			nextTaskIdx = 0;
 			chargeBeforeNextTask = battery.getCharge();
-		} else { // STARTED
+			// STARTED, but the vehicle is out of service time
```

Comment:
I think this comment should be moved one line down.

Reply:
Is it really `STARTED, but the vehicle is out of service time`? Then you would not get `failIfNotStarted`?